ballerina/soap.bal

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/mime;
+
+# Soap version.
+public type SoapVersion SOAP11 | SOAP12;
```

Comment:
We can define an enum here
```bal
public enum SoapVersion {
    SOAP11,
    SOAP12
}
```

Reply:
Updated in [a3068ee](https://github.com/ballerina-platform/module-ballerina-soap/pull/82/commits/a3068ee67d1a4d7667ae0023565f0bc525eae7af)